packages/eas-cli/src/build/build.ts

```diff
@@ -274,7 +282,11 @@ async function uploadProjectAsync<TPlatform extends Platform>(
             completedMessage: (duration: string) => `Uploaded to EAS ${chalk.dim(duration)}`,
           })
         );
-        return bucketKey;
+        const { metadataLocation } = await uploadMetadataFileAsync<TPlatform>(projectTarball, ctx);
+        if (metadataLocation) {
+          return { bucketKey, metadataLocation };
```

Comment:
I remain on the position the fact we have different names (key, location) for the same thing in the same object and the newer name is of less specific meaning is wrong. This comment has been left unanswered for a couple iterations now (first mentioned [here](https://github.com/expo/eas-cli/pull/2149#discussion_r1432764804)) and the conversation marked as resolved twice which is confusing for me.